build.gradle.kts

```diff
@@ -197,20 +192,24 @@ tasks.register("downloadDependencies") {
 	}
 }
 
-val getLoc () {
-	int linesOfCode = 0
-	int filesCount = 0
-	project.sourceSets.main.allSource.srcDirs.each {
-		File dir ->
-		if (dir.isDirectory()) {
-			dir.eachFileRecurse {
-				File file ->
-				if (file.isFile()) {
-					file.eachLine(e -> linesOfCode++)
-					filesCount++
+tasks.register("getLoc") {
```

Comment:
I don't know if this is right, but this shouldn't be a task (I think), but rather a helper method that is used in the build task

Reply:
ignore for know